packages/generate-newsletter/index.js

```diff
@@ -43,12 +43,13 @@ const askQuestions = async () => {
 askQuestions().then(userResponse => {
   console.log('\n Generating pls wait... \n ')
 
-  extractSheets({ spreadsheetKey, credentials }, (err, data) => {
+  extractSheets({ spreadsheetKey, credentials, sheetsToExtract: ['latest'] }, (err, data) => {
```

Comment:
In-order to take from first tab we have to give this ?

Reply:
yeah... remember there was an issue: sometimes the html wont get generated. This is because the order of the keys cannot be trusted. So, we cannot do `data[0]` to get the latest content.

So, better to keep a spreadsheet named 'latest' for the one we need to generate.